### PR TITLE
Added DBML Serializer (just hacked it dirty in, better than nothing)

### PR DESCRIPTION
**Pull request by Kaev**
Quick and dirty implementation for DBML export to generate this: https://dbdocs.io/Kaev/DBC_3_3_5_12340
Only exports for a single build that you need to provide as a new argument.

### CODE REVIEW

File: code/C#/DBDefsLib/Structs.cs [hunk 1]
```diff
@@ -2,42 +2,39 @@
 
 namespace DBDefsLib
 {
-    public class Structs
```
Comment (bloerwald) [inline code/C#/DBDefsLib/Structs.cs:5]: This is a library, so external users will be broken by these changes. Even if the name is technically wrong, I'd avoid the rename and maybe add an alias only.

Comment (Kaev) [reply code/C#/DBDefsLib/Structs.cs]: I've reverted that

**Comment by Marlamin** [comment]
Not sure why there's a bunch of Structs. related changes in other files unrelated to this functionality, seems unnecessary, don't think it'll break anything but might be better to just not do those.

**Comment by Kaev** [comment]
> Not sure why there's a bunch of Structs. related changes in other files unrelated to this functionality, seems unnecessary, don't think it'll break anything but might be better to just not do those.

Rider replaced that for me when i did the initial (reverted) namespace change. I reverted these. :)

**Comment by Kaev** [comment]
I merged #157 and #158 into this so only merge after they got merged

**Pull request merged**